packages/template-retail-react-app/app/utils/product-utils.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
```

Comment:
```suggestion
 * Copyright (c) 2023, Salesforce, Inc.
```
How'd you end up with 2023 and salesforce.com? Weird.